.github/workflows/e2e_lighthouse_performance_acceptance_tests.yml

```diff
@@ -125,66 +74,40 @@ jobs:
             requirements_dev.in
             requirements_dev.txt
       - uses: ./.github/actions/merge
-      - name: Cache node modules
-        uses: actions/cache@v3
-        env:
-          cache-name: cache-node-modules
-        with:
-          path: /home/runner/work/oppia/yarn_cache
-          key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('yarn.lock') }}
-          restore-keys: |
-            ${{ runner.os }}-build-${{ env.cache-name }}-
-            ${{ runner.os }}-build-
-            ${{ runner.os }}-
-      - name: Attempt to download build files artifact
-        id: download_artifact_1
-        uses: actions/download-artifact@v3
-        continue-on-error: true
-        with:
-          name: build_files
-          path: /home/runner/work/oppia/
-      - name: Unzip build files
-        if: steps.download_artifact_1.outcome != 'failure'
+      - name: Install Python deps
         run: |
-          ls -la
-          unzip build_files.zip
-          rm build_files.zip
-          echo "Files in ./:"
-          ls -la .
-          echo "Files in oppia_tools:"
-          ls -la oppia_tools
-          echo "Files in oppia:"
-          ls -la oppia
-          echo "Files in build:"
-          ls -la oppia/build
-          echo "Files in third_party:"
-          ls -la oppia/third_party
-        working-directory: /home/runner/work/oppia
-        shell: bash
-      - uses: ./.github/actions/install-oppia-dependencies
+          pip install --require-hashes --no-deps -r requirements.txt
+          pip install --require-hashes --no-deps -r requirements_dev.txt
```

Comment:
Why are we installing Python dependencies outside the Docker?

Reply:
https://github.com/oppia/oppia/pull/18999#discussion_r1371145039
(please check above thread)